CHANGELOG.md

```diff
@@ -7,6 +7,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- Fallible versions of the `OutputPin` and `StatefulOutputPin` traits have been added.
+  These are versions where the methods return a `Result` type as setting an output pin
+  could potentially fail.
+  The current trait versions are now marked as deprecated. The fallible version
```

Comment:
Should we be marking the new traits with `unproven` too? (I'm not sure if there is a precedent, seems ok either way)

Reply:
`StatefulOutputPin` is already marked as `unproven` and its interface will not change if not activating `use-fallible-digital-traits`. The only fallible trait one not behind the `unproven` feature is the fallible `OutputPin`.